plugins/saveactions.c

```diff
@@ -635,7 +1237,46 @@ static void checkbox_toggled_cb(GtkToggleButton *tb, gpointer data)
 			break;
 		}
 	}
+}
+
+
+static void radio_toggled_cb(GtkRadioButton *rb, gpointer data)
+{
+	gboolean enable = gtk_toggle_button_get_active(rb);
```

Comment:
Here a cast is missng:
```suggestion
	gboolean enable = gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(rb));
```